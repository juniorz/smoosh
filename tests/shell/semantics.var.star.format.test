sp="s p  aces"
tn=$(printf '%b' 'and\ttabs\n and newlines')
set -- a "$sp" b c "$tn"
IFS=": "
printf '<%s>\n' "${var=$*}"
unset var
unset IFS
printf '<%s>\n' "${var=$*}"
