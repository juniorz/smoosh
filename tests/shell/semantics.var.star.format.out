<a:s p  aces:b:c:and	tabs
 and newlines>
<a s p  aces b c and	tabs
 and newlines>
